projects/vdk-core/cicd/build.sh

```diff
@@ -21,7 +21,7 @@ cd ..
 export PIP_EXTRA_INDEX_URL=${PIP_EXTRA_INDEX_URL:-https://test.pypi.org/simple/}
 
 echo "Update pip to newest version"
-pip install -U pip
+pip install -U pip==22.0
```

Comment:
write a comment on why it's pinned. And create a ticket to unpinn it.

Reply:
Removed.